src/renderer/components/query-result-table.jsx

```diff
@@ -336,7 +336,8 @@ export default class QueryResultTable extends Component {
    * It gives a better UX since the column adapts better to the table width.
    */
   resolveCellWidth(fieldName, fields, rows) {
-    const font = '14px \'Lato\', \'Helvetica Neue\', Arial, Helvetica, sans-serif';
+    const { customFont } = this.props.config.data.customFont;
```

Comment:
@maxcnunes  This line here actually breaks the GUI on a fresh checkout as `customFont` doesn't exist.

Reply:
Good catch, are able to fix it? 🙏